recipe/meta.yaml

```diff
@@ -11,43 +11,49 @@ source:
   url: https://github.com/mysql/mysql-connector-python/archive/{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-    - 0001-Typecast-password-to-const-char-explicitly-py3.patch
-    - 0002-Cast-uint-to-enum-mysql_enum_shutdown_level-explicit.patch
-    - 0003-Don-t-error-out-if-get_git_info-fails.patch
+    - patches/0001-Typecast-password-to-const-char-explicitly-py3.patch
+    - patches/0002-Fix-location-for-searching-mysql-headers-and-libs-on.patch
+    - patches/0003-Fix-indentation-for-common-flags.patch
+    - patches/0004-Revert-Fix-MacOS-wheels-platform-tag.patch
+    - patches/0005-Replace-google-protobuf-string-with-std-string.patch
+    - patches/0006-Delete-openssl-vendor-ed-libs-only-if-provided.patch
 
 build:
   number: 1
   # VS 2008 is not supported for libprotobuf
-  skip: True  # [win and py2k]
-  script: python setup.py install --with-protobuf-include-dir=$PREFIX/include --with-protobuf-lib-dir=$PREFIX/lib --with-protoc=$PREFIX/bin/protoc --with-mysql-capi=$PREFIX  # [unix]
-  script: python setup.py install --with-protobuf-include-dir=%LIBRARY_INC% --with-protobuf-lib-dir=%LIBRARY_LIB% --with-protoc=%LIBRARY_BIN%/protoc.exe --with-mysql-capi=%LIBRARY_PREFIX% --extra-compile-args=/DPROTOBUF_USE_DLLS  # [win]
-
+  skip: True  # [py<38]
+  ignore_run_exports:
+    - libprotobuf
+    - libstdcxx-ng
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - patch       # [not win]
     - m2-patch    # [win]
- 
+    - pkg-config
   host:
     - python
+    - pip
     - setuptools
-    - mysql-connector-c
-    - libprotobuf
-    - protobuf >=3.0.0
-
+    - mysql {{ version }}  # [not win]
+    - protobuf 4.25.3
+    - libprotobuf 4.25.3
   run:
     - python
-    - protobuf >=3.0.0
+    - mysql  # [not win]
+    - protobuf
```

Comment:
I don't think it's needed at runtime. I'm not even sure that it's required at build time to be honest.

Reply:
There's an optional C Extension in `mysql-connector-python` that do require those according to their docs:
https://dev.mysql.com/doc/connector-python/en/connector-python-installation-source.html
Since we supported the C Extension previously, I've decided to keep it that way, except for windows, where it cannot be done without `mysql` or `mysql-connector-c`